src/ui/flutter_app/lib/shared/constants.dart

```diff
@@ -24,7 +24,7 @@ class Constants {
   static const String authorAccount = "calcitem";
   static const String projectName = "Sanmill";
   static String projectNameLower = projectName.toLowerCase();
-  static const String recipients = "$authorAccount@outlook.com";
+  static const List<String> recipients = ["$authorAccount@outlook.com"];
```

Comment:
It is recommended to change to RecipientsEmail, because all authors are used in Outlook mailboxes.

That is to say, move `@outlook.com` to constants together.

Reply:
I don't get what you mean :(

Reply:
I will modify it later myself.

Reply:
Ok, I'll mark this one as WIP so you can add your changes...

Reply:
No problem. You can merge it.